tests/valid/draft-v3-features.text

```diff
@@ -458,7 +458,6 @@ Table of Contents
    |  and computer keyboards, and in other applications involving all of
    |  the letters in the English alphabet.  Owing to its brevity and
    |  coherence, it has become widely known.
-   |  
```

Comment:
Is this loss of whitespace intentional? I could see it introducing confusion in existing documents.

Reply:
I mistakingly thought non-blockquote behaviour was to have no new line there.
This has been corrected with dac1eca70f404ad92ae28c27d12bcb89d8e44371
I don't see any adverse effect from that change.